draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1156,27 +1164,8 @@ If the certificate is logged, an SCT MUST be issued. Logging the certificate is
 useful, because monitors ({{monitor}}) can then detect these encoding errors,
 which may be accepted by some TLS clients.
 
-## Add PreCertChain to Log    {#add-pre-chain}
-
-POST https://\<log server>/ct/v2/add-pre-chain
-
-Inputs:
-
-: precertificate:
-  : The base64 encoded precertificate.
-
-  chain:
-  : An array of base64 encoded CA certificates. The first element is the signer
-    of the precertificate; the second certifies the first and so on to the last,
-    which either is, or is certified by, an accepted trust anchor.
-
-Outputs:
-
-: sct:
-  : A base64 encoded `TransItem` of type `precert_sct_v2`, signed by this log,
-    that corresponds to the submitted precertificate.
-
-Errors are the same as in {{add-chain}}.
+A log MUST reject a submission where both the `precertificate` and `certificate`
+parameters were specified and return the `ambiguous submission` error.
```

Comment:
What happens if neither `precertificate` nor `certificate` is specified?  Perhaps add a "bad submission" error code to handle that case?